files/ru/web/css/css_positioning/understanding_z_index/the_stacking_context/index.md

```diff
@@ -38,7 +38,7 @@ translation_of: Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_conte
 
 ## Пример
 
-![Example of stacking rules modified using z-index](/@api/deki/files/913/=Understanding_zindex_04.png)
+![Example of stacking rules modified using z-index](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context/understanding_zindex_04.png)
```

Comment:
Hi @rohini-ranjanR. You can just use `understanding_zindex_04.png` as the platform would automaticly fallback to `en-US`

```suggestion
![Example of stacking rules modified using z-index](understanding_zindex_04.png)
```

Reply:
Hi, @yin1999 I accidentally delete the forked repository related to this PR. Now, I am not getting any option to edit the commit or make suggested changes.
So, I am creating another pull request with suggested changes.

Reply:
Thank you. Let's close this one.